meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk.bb

```diff
@@ -97,6 +98,7 @@ do_install () {
     sed -i -e 's:@SUPERVISOR_REPOSITORY@:${SUPERVISOR_REPOSITORY}:g' ${D}${sysconfdir}/resin-supervisor/supervisor.conf
     sed -i -e 's:@TARGET_TAG@:${TARGET_TAG}:g' ${D}${sysconfdir}/resin-supervisor/supervisor.conf
```

Comment:
This line does not do anything as PARGET_TAG is not defined in the sample file this command is trying to perform a sed on.
Shall we also remove this line in this PR @telphan @agherzan ?